lib/testing/testDataCheck.hpp

```diff
@@ -102,7 +102,7 @@ void testDataCheck<A_Type>::main_thread() {
             } else { // N2 numbers are int
                 // INFO("Checking non float numbers-----------");
                 if (first_value != second_value) {
-                    if (num_errors++ < 10000)
+                    if (num_errors++ < 20)
```

Comment:
We've changed this value enough times that it should be a config option with a default of something like `100`